transcendence-app/src/app.module.ts

```diff
@@ -10,7 +13,29 @@ import { validate } from './config/env.validation';
       cache: true,
       validate,
     }),
-    OAuth42Module,
+    AuthModule,
   ],
 })
-export class AppModule {}
+export class AppModule {
+  constructor(private configService: ConfigService) {}
+
+  configure(consumer: MiddlewareConsumer) {
+    const MemcachedStore = ConnectMemcached(session);
+
+    consumer
+      .apply(
+        session({
+          secret: this.configService.get('SESSION_SECRET') as string,
+          resave: false,
+          saveUninitialized: false,
+          store: new MemcachedStore({
+            hosts: ['memcached:11211'],
```

Comment:
Why is it not a good idea in this context to store this string in an environmental variable?